components/page-header/src/index.js

```diff
@@ -33,13 +36,13 @@ const NavigationContainer = styled('div')({
  * ### Usage
  *
  * Default example
- * 
+ *
  * ```jsx
  * import { Link } from 'react-router-dom';
- * import PageHeader, { asLogoAnchor, asNavAnchor } from '@govuk-frederic/page-header';
- * 
- * const LogoLink = asLogoAnchor(Link);
- * const NavLink = asNavAnchor(Link);
+ * import PageHeader from '@govuk-frederic/page-header';
+ *
+ * const LogoLink = asNavLink()(PageHeader.LogoAnchor);
```

Comment:
for this example to work, you'd also need to `import asNavLink from 'as-nav-link';`

Reply:
done, thanks